### PR TITLE
Add Zoom links to the description attribute

### PR DESCRIPTION
**Pull request by InessaPawson**
Adding Zoom links to the description attribute since the links in the url attribute aren't visible to subscribers of the calendar.

### CODE REVIEW

File: calendars/numpy.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ events:
 
       To add to the meeting agenda the topics you’d like to discuss,
       follow the link: https://hackmd.io/76o-IxCjQX2mOXO_wwkcpg
+
+      Join us via Zoom: https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
+
```
Comment (bsipocz) [inline calendars/numpy.yaml:13]:
```suggestion

```

File: calendars/numpy.yaml [hunk 2]
```diff
@@ -24,6 +27,9 @@ events:
 
       Please notify us of an issue or PR that you’d like to have triaged/reviewed by adding
       a GitHub link to it in the meeting agenda: https://hackmd.io/68i_JvOYQfy9ERiHgXMPvg
+
+      Join us via Zoom: https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
+
```
Comment (bsipocz) [inline calendars/numpy.yaml:32]:
```suggestion

```

File: calendars/numpy.yaml [hunk 3]
```diff
@@ -41,6 +47,9 @@ events:
 
       To add to the meeting agenda the topics you'd like to discuss,
       follow the link: https://hackmd.io/oB_boakvRqKR-_2jRV-Qjg
+
+      Join us via Zoom: https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
+
```
Comment (bsipocz) [inline calendars/numpy.yaml:52]:
```suggestion

      Join us via Zoom: https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
 
```

File: calendars/numpy.yaml [hunk 4]
```diff
@@ -58,6 +67,9 @@ events:
...
```
Comment (bsipocz) [inline calendars/numpy.yaml:72]:
```suggestion

```

File: calendars/numpy.yaml [hunk 5]
```diff
@@ -73,6 +85,9 @@ events:
 
       To add to the meeting agenda the topics you’d like to discuss,
       follow the link: https://hackmd.io/3f3otyyuTte3FU9y3QzsLg
+
+      Join us via Zoom: https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
+
```
Comment (bsipocz) [inline calendars/numpy.yaml:90]:
```suggestion

```

File: calendars/numpy.yaml [hunk 6]
```diff
@@ -88,6 +103,9 @@ events:
...
```
Comment (bsipocz) [inline calendars/numpy.yaml:108]:
```suggestion

```

**Review by bsipocz — APPROVED** [review]
To make the linter happy

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**